deps.edn

```diff
@@ -12,5 +12,9 @@
         rewrite-clj/rewrite-clj {:mvn/version "1.1.47"}}
  :aliases
  {:test {:extra-paths ["test"]
-         :extra-deps {clj-commons/conch {:mvn/version "0.9.2"}}}}
+         :extra-deps {clj-commons/conch {:mvn/version "0.9.2"}}}
+  :native {:extra-deps {com.github.clj-easy/graal-build-time {:mvn/version "0.1.4"}
+                        org.graalvm.nativeimage/svm {:mvn/version "21.3.0"}}}
```

Comment:
I think it's ok to remove the `graal-build-time` as far as you want build it only with nix, since `clj-nix` already adds it to the classpath: 
https://github.com/jlesquembre/clj-nix/blob/3dffacf858ec9eb882d06695734b5b4300aa9448/pkgs/mkGraalBin.nix#L76-L80

What is `org.graalvm.nativeimage/svm` doing? Maybe makes sense that `clj-nix` adds `svm` too?

Reply:
The `org.graalvm.nativeimage/svm` was only needed for the javac compile of one .java file that's in the jet project.  I actually don't know what it's for ... :)

I am not adding this `:native` alias to the clojure compile in `mkCljBin` though.  I probably shouldn't have called this alias `:native` - it's really `:javac`.